source/standards/tracking-dependencies.html.md.erb

```diff
@@ -14,27 +14,66 @@ You can read more about [managing software dependencies in the Service Manual][]
+
 ## Monitoring for vulnerabilities
 
-### Ruby - GOV.UK Gem Security Checker
+You should monitor for potential vulnerabilities in every layer of your technology stack. This is not straightforward but tooling exists to help. The Service Manual provides [guidance on browser technology, tooling and mailing lists you can subscribe to][].
+
+Most of the tooling that helps you stay on top of dependency updates will also highlight vulnerabilities. Additional tooling includes:
+
+* [GitHub security alerts][] - This can be configured per repository in the settings tab of the GitHub UI. When GitHub discovers or is informed about a vulnerability, it will email an alert to the repository owner and users with admin access
+* [Snyk][] - Snyk is capable of detecting vulnerabilities in a variety of languages including all the [GDS supported programming languages][]. Snyk can be configured to raise PRs, email regular reports and alert you when new vulnerabilities are detected
+
+### Docker images
+
+Always use [official base images][]. Docker Hub regularly scans official images and the results can be viewed by logging into Docker Hub. If you do not regularly update your base image, you must ensure a manual process exists to monitor and prioritise fixes for vulnerabilities detected.
 
-You should use the [GOV.UK Gem Security Checker][] alongside your regular code checks. This will help your team move any code vulnerabilities into the team’s work backlog. You can then:
+The Digital Marketplace team uses [Snyk container vulnerability management tooling][] to regularly scan base images.
 
-* prioritise fixes relative to other project work
-* address vulnerabilities in private before making the fix public
+## Minimising what you need to monitor
 
-### Snyk
+Minimise the things you need to monitor by minimising dependencies where practically possible. For example, if running containers, make sure you pick the smallest base images. 
 
-[Snyk][] is a dependency management tool which can [monitor code dependencies][] for security vulnerabilities. Using Snyk allows you to automatically check GitHub pull requests (PRs) for vulnerable dependencies and potentially create a pull request to fix them. It also supports the other languages that GDS uses, including Python, Java and JavaScript.
+Also consider managed solutions where possible. For example:
 
-> The GDS Way does not mandate the use of Snyk: please contribute any experience you have of other dependency management tools to this document.
+* Use [GOV.UK PaaS buildpacks][] so you do not have to monitor for Operating System (OS), runtime or programming language vulnerabilities
+* Consider managed container orchestration technology such as [AWS Fargate][], [Google Kubernetes Engine][] or similar so you do not have to monitor for vulnerabilities in your OS or control plane
```

Comment:
Given our current platform of choice is AWS, I don't think we should be mentioning Google's offering. I think "such as AWS Fargate or similar" is good enough.

Reply:
Have changed, Thanks Tim!